tensorflow_privacy/privacy/membership_inference_attack/seq2seq_mia.py

```diff
@@ -204,18 +276,24 @@ def create_seq2seq_attacker_data(attack_input_data: Seq2SeqAttackInputData,
     model_selection.train_test_split(
         features_all, labels_all, test_size=test_fraction, stratify=labels_all)
 
+  # Populate fields of privacy report metadata
+  privacy_report_metadata.loss_train = loss_train
```

Comment:
Can this be None?

Reply:
Did you mean if privacy_report_metadata can be None? In line [313](https://github.com/tensorflow/privacy/pull/144/files/46bee91cda7e68f88a6336d9f299641eaba818f9..eb215072bcc18f202c69ed51508185e9bfa9744e#diff-acd940c715bb3abd65068035e671ca82e8a6e1287082ce7e1a638728f417d383R313) I perform a check for this and initialize privacy_report_metadata if it is None. I can move it to this line if it would make the code more readable

Reply:
I see. what confuses me is that the parameter in this method has a default value of None, but there are no further checks after that. So maybe initialize to a default of PrivacyReportMetadata? or throw an error if it's None? Or move the check.

Reply:
Oh, I can see how that is confusing. I will add a check